tools/profiles/debug-size.json

```diff
@@ -0,0 +1,44 @@
+{
+    "GCC_ARM": {
+        "common": ["-c", "-Wall", "-Wextra",
+                   "-Wno-unused-parameter", "-Wno-missing-field-initializers",
+                   "-fmessage-length=0", "-fno-exceptions", "-fno-builtin",
+                   "-ffunction-sections", "-fdata-sections", "-funsigned-char",
+                   "-MMD", "-fno-delete-null-pointer-checks",
+                   "-fomit-frame-pointer", "-O1", "-g"],
```

Comment:
I believe the reason debug symbols aren't turn on in all configurations is because this increases build sizes too much for testing and the online IDE. @screamerbg, @bridadan or @0xc0170 can you confirm this? If debug symbols could be turned on/off independent of the profile the each of the profiles wouldn't need have a corresponding "debug-*" profile to add symbols. This would be especially useful if the binary output doesn't change (which it shouldn't).